timesketch/lib/datastores/elastic.py

```diff
@@ -328,6 +335,7 @@ def set_label(
             script_name = u'toggle_label'
         script = {
             u'script': {
+                u'lang': u'groovy',
```

Comment:
Groovy is still supported. I want to do some more testing first and update this in a later PR.